### ISSUE TITLE
The chaincode container builded failed and return 137

### ISSUE THREAD

**Issue by teddycode** [blewater/pifabric#1]
host：huawei kunpeng ecs with 4 core 8 gb arm64 archtecture。
step：./initcc.sh
error message：
2020-06-23 01:38:39.242 UTC [chaincodeCmd] InitCmdFactory -> INFO 001 Retrieved channel (mychannel) orderer endpoint: orderer.example.com:7050
2020-06-23 01:38:39.243 UTC [chaincodeCmd] checkChaincodeCmdParams -> INFO 002 Using default escc
2020-06-23 01:38:39.243 UTC [chaincodeCmd] checkChaincodeCmdParams -> INFO 003 Using default vscc
Error: could not assemble transaction, err proposal response was not successful, error code 500, msg error starting container: error starting container: Failed to generate platform-specific docker build: Error returned from build: 137 "Killed

another error message like this：

$ ./initcc.sh
test, 1000
2020-06-23 01:31:10.325 UTC [chaincodeCmd] InitCmdFactory -> INFO 001 Retrieved channel (mychannel) orderer endpoint: orderer.example.com:7050
2020-06-23 01:31:10.327 UTC [chaincodeCmd] checkChaincodeCmdParams -> INFO 002 Using default escc
2020-06-23 01:31:10.327 UTC [chaincodeCmd] checkChaincodeCmdParams -> INFO 003 Using default vscc
Error: could not assemble transaction, err proposal response was not successful, error code 500, msg error starting container: error starting container: Failed to generate platform-specific docker build: Error returned from build: 1 "go tool cgo: signal: killed
go build github.com/hyperledger/fabric/common/metadata: /opt/go/pkg/tool/linux_arm64/compile: exit status 1
go build strconv: /opt/go/pkg/tool/linux_arm64/compile: fork/exec /opt/go/pkg/tool/linux_arm64/compile: argument list too long

addtional： start byfn also failed！

**Comment by xingxindrst** [comment]
> 

Have you solved this problem?I'm having the same problem.

**Comment by xingxindrst** [comment]
> host：huawei kunpeng ecs with 4 core 8 gb arm64 archtecture。 step：./initcc.sh error message： 2020-06-23 01:38:39.242 UTC [chaincodeCmd] InitCmdFactory -> INFO 001 Retrieved channel (mychannel) orderer endpoint: orderer.example.com:7050 2020-06-23 01:38:39.243 UTC [chaincodeCmd] checkChaincodeCmdParams -> INFO 002 Using default escc 2020-06-23 01:38:39.243 UTC [chaincodeCmd] checkChaincodeCmdParams -> INFO 003 Using default vscc Error: could not assemble transaction, err proposal response was not successful, error code 500, msg error starting container: error starting container: Failed to generate platform-specific docker build: Error returned from build: 137 "Killed
> 
> another error message like this：
> 
> $ ./initcc.sh test, 1000 2020-06-23 01:31:10.325 UTC [chaincodeCmd] InitCmdFactory -> INFO 001 Retrieved channel (mychannel) orderer endpoint: orderer.example.com:7050 2020-06-23 01:31:10.327 UTC [chaincodeCmd] checkChaincodeCmdParams -> INFO 002 Using default escc 2020-06-23 01:31:10.327 UTC [chaincodeCmd] checkChaincodeCmdParams -> INFO 003 Using default vscc Error: could not assemble transaction, err proposal response was not successful, error code 500, msg error starting container: error starting container: Failed to generate platform-specific docker build: Error returned from build: 1 "go tool cgo: signal: killed go build github.com/hyperledger/fabric/common/metadata: /opt/go/pkg/tool/linux_arm64/compile: exit status 1 go build strconv: /opt/go/pkg/tool/linux_arm64/compile: fork/exec /opt/go/pkg/tool/linux_arm64/compile: argument list too long
> 
> addtional： start byfn also failed！



> host：huawei kunpeng ecs with 4 core 8 gb arm64 archtecture。 step：./initcc.sh error message： 2020-06-23 01:38:39.242 UTC [chaincodeCmd] InitCmdFactory -> INFO 001 Retrieved channel (mychannel) orderer endpoint: orderer.example.com:7050 2020-06-23 01:38:39.243 UTC [chaincodeCmd] checkChaincodeCmdParams -> INFO 002 Using default escc 2020-06-23 01:38:39.243 UTC [chaincodeCmd] checkChaincodeCmdParams -> INFO 003 Using default vscc Error: could not assemble transaction, err proposal response was not successful, error code 500, msg error starting container: error starting container: Failed to generate platform-specific docker build: Error returned from build: 137 "Killed
> 
> another error message like this：
> 
> $ ./initcc.sh test, 1000 2020-06-23 01:31:10.325 UTC [chaincodeCmd] InitCmdFactory -> INFO 001 Retrieved channel (mychannel) orderer endpoint: orderer.example.com:7050 2020-06-23 01:31:10.327 UTC [chaincodeCmd] checkChaincodeCmdParams -> INFO 002 Using default escc 2020-06-23 01:31:10.327 UTC [chaincodeCmd] checkChaincodeCmdParams -> INFO 003 Using default vscc Error: could not assemble transaction, err proposal response was not successful, error code 500, msg error starting container: error starting container: Failed to generate platform-specific docker build: Error returned from build: 1 "go tool cgo: signal: killed go build github.com/hyperledger/fabric/common/metadata: /opt/go/pkg/tool/linux_arm64/compile: exit status 1 go build strconv: /opt/go/pkg/tool/linux_arm64/compile: fork/exec /opt/go/pkg/tool/linux_arm64/compile: argument list too long
> 
> addtional： start byfn also failed！

Have you solved this problem?I'm having the same problem.

**Comment by blewater** [comment]
No, that's an old project, and I haven't looked at the recent build configuration and language toolchain.
